### ISSUE TITLE
French 3 Streams Are Wrong? 

### ISSUE THREAD

**Issue by heaavypetal** [Trontor/lookahead#19]
Hey, just trying to do my subjects and the options for French 3 which come up on Lookahead are totally different to on Allocate. 
<img width="805" alt="Screen Shot 2021-02-06 at 1 20 52 pm" src="https://user-images.githubusercontent.com/78629116/107106176-6d120e80-687e-11eb-9903-c26192c67b5b.png">
<img width="1440" alt="Screen Shot 2021-02-06 at 1 20 31 pm" src="https://user-images.githubusercontent.com/78629116/107106178-6e433b80-687e-11eb-9d21-a851ba9e293f.png">

**Comment by manu354** [comment]
Lookahead gets its class info from SWS: https://sws.unimelb.edu.au/2021/Reports/List.aspx?objects=FREN10001&weeks=1-52&days=1-7&periods=1-56&template=module_by_group_list

![image](https://user-images.githubusercontent.com/14151679/107107855-dfd3b780-6887-11eb-926b-ce43900963d1.png)

As far as I know, there is no way to get the class data from Allocate, so if there is a mismatch between the classes on Allocate and SWS (as I believe is the case here) there is no way to know or fix it. 

So please correct me if I am wrong, but I think this would have to be addressed by the university, by updating the SWS entries.

**Comment by Trontor** [comment]
That's correct @manu354, SWS is lookahead's only source of truth. Unfortunately it seems like the data fed into Allocate+ is mismatched. Until the university is kind enough to allow us access to the same data set there's not much we can do. Maybe a feature where students can rate the subject data as accurate or inaccurate would help to mitigate unintentionally misleading people.

Regardless, thank you for reporting this @heaavypetal -- it's always good to see people actively reporting issues and inconsistencies :)
